src/client.ts

```diff
@@ -235,6 +236,11 @@ export class StreamChat<
     }
 
     this.axiosInstance = axios.create(this.options);
+    if (this.options.safari || isSafari()) {
```

Comment:
Whats the purpose of `isSafari()`? I can see that it always returns false

Reply:
We released this for a beta so that we can test with customer. If works, we will implement detection. Right now, it bypasses with an option for testing.